src/Manifolds.jl

```diff
@@ -989,6 +989,8 @@ export adjoint_action,
     identity_element!,
     inv,
     inv!,
+    inv_diff,
```

Comment:
would `group_inv_diff`  or even `group_inverse_diff` maybe be a better name? Since we use inv already the first might be better...

Reply:
OK, `group_inv_diff` sounds good.

Reply:
Hm, maybe the current is also fine since the original function is called `inv`,